esp32_uwb_publisher:
  ros__parameters:
    port: '/dev/ttyUWB'
    baud_rate: 115200
    timer_rate: 100.0
    pub_rate: 10.0
    anchor_id: [1781, 1782, 1783, 1784]

# echo 'SUBSYSTEM=="tty", ATTRS{idVendor}=="10c4", ATTRS{idProduct}=="ea60", MODE="0666", SYMLINK+="ttyUWB"' | sudo tee /etc/udev/rules.d/99-esp32-uwb-test.rules
# sudo udevadm control --reload && sudo udevadm trigger